src/modules/rover_ackermann/RoverAckermannGuidance/RoverAckermannGuidance.hpp

```diff
@@ -57,6 +56,7 @@
 #include <lib/pid/pid.h>
 
 using namespace matrix;
+static constexpr uint8_t NAVIGATION_STATE_AUTO_RTL = 5;
```

Comment:
Can you not use `vehicle_status_s::NAVIGATION_STATE_AUTO_RTL` instead of re-defining it here?

Reply:
This definition doesn`t exist in the context of this file because I don't include `vehicle_status` here since I don`t use it.

Reply:
Well yes you use it now here, otherwise you wouldn't need to re-define the constant. I would include vehicle_status_s.

Reply:
Included in https://github.com/PX4/PX4-Autopilot/pull/23310/commits/6730d226787792ece02c3af60d36a6d54f33894c